fcl/detsim/detsim_2d_icarus_shiftForTrigger.fcl

```diff
@@ -0,0 +1,98 @@
+#include "trigger_emulation_icarus.fcl"
+#include "services_icarus_simulation.fcl"
+#include "detsimmodules_wirecell_ICARUS.fcl"
+#include "opdetsim_pmt_icarus.fcl"
+#include "crtsimmodules_icarus.fcl"
+#include "rootoutput_icarus.fcl"
+
+process_name: DetSim
+
+services: {
+  @table::icarus_detsim_services
+} # services
+
+
+physics: {
+
+  producers: {
+    crtdaq:         @local::icarus_crtsim
+     opdaq:         @local::icarus_simpmt
+       daq:         @local::icarus_simwire_wirecell
+
+    @table::icarus_standard_triggersim.producers
+    pmtfixedthrinit:       @local::icarus_pmtdiscriminatethr_fixed_MC
+
+       shiftIonization:
+       {
+         module_type: AdjustSimForTrigger
+         InputBeamGateInfoLabel: "beamgate"
+         InputSimEnergyDepositLabel: "ionization"
+         InputSimPhotonsLabel: "largeant"
+         InputTriggerLabel: "emuTriggerUnshifted"
+         InputWaveformLabel: "opdaq"
+         InputAuxDetSimChannelLabel: "genericcrt"
+         AdditionalOffset: 0.0 # [us] Additional offset to apply to the time shift
+         ShiftSimEnergyDeposits: true
+         ShiftSimPhotons: true
+         ShiftWaveforms: true
+         ShiftBeamGateInfo: true
+       }
+
+       rns:         { module_type: "RandomNumberSaver" }
+  } # producers
+
+
+  simulate: [
+      rns,
+      opdaq,
+      pmtfixedthrinit,
+      pmtlvdsgatesinit,
+      pmttriggerwindowsinit,
+      triggersimgatesinit,
+      emuTriggerUnshifted,
+      shiftIonization,
+      daq,
+      crtdaq
+  ]
+
+  # define the output stream, there could be more than one if using filters
+  stream:  [ rootoutput ]
+
+} # physics
+
+physics.producers.pmtlvdsgatesinit: {
+  @table::physics.producers.pmtlvdsgates
+  TriggerGatesTag: "pmtfixedthrinit"
+}
+
+physics.producers.pmttriggerwindowsinit: {
+  @table::physics.producers.pmttriggerwindows
+  TriggerGatesTag: "pmtlvdsgatesinit"
+}
+
+physics.producers.triggersimgatesinit: {
+  @table::physics.producers.triggersimgates
+  BeamGateTag: "beamgate"
+}
+
+physics.producers.crtdaq.G4ModuleLabel: "shiftIonization"
+
+physics.producers.emuTriggerUnshifted: {
+  @table::physics.producers.emuTrigger
+  BeamGates: "triggersimgatesinit"
+  TriggerGatesTag: "pmttriggerwindowsinit"
+}
+
+outputs: {
+  rootoutput: {
+    @table::icarus_rootoutput
+    outputCommands: [
+      "keep *",
+      "drop *_ionization_*_*",
+      "drop *_pmtlvdsgatesinit_*_*",
+      "drop *_pmtfixedthrinit_*_*",
+      "drop *_pmttriggerwindowsinit_*_*",
+      "drop *_triggersimgatesinit_*_*"
+      ]
+  }
+}
```

Comment:
Does this fhicl need something like ```physics.producers.daq.wcls_main.configs: ["pgrapher/experiment/icarus/wcls-multitpc-sim-drift-simchannel-shifted.jsonnet"]```
that is in the other files?

I guess no since the shift was tested? Or maybe just there is a local change not on the remote. (( Just checking while quickly looking through the changes! I can also think if I'm missing something here. ))

Reply:
100% this is an oversight. Will push an update promptly.

Reply:
Done: https://github.com/SBNSoftware/icaruscode/pull/673/commits/dca13e09946c903191ed046e3e382629effe95fb

Reply:
Cool: thanks!